joblib/_parallel_backends.py

```diff
@@ -131,14 +138,14 @@ def effective_n_jobs(self, n_jobs):
 
     def terminate(self):
         """Shutdown the process or thread pool"""
-        if self._pool is not None:
-            self._pool.close()
-            self._pool.terminate()  # terminate does a join()
-            self._pool = None
+        if self._workers is not None:
+            self._workers.close()
+            self._workers.terminate()  # terminate does a join()
+            self._workers = None
```

Comment:
Because 3rd party backends might want to reuse this mixin class I would rather not rename that private attribute.